docs/supported_platforms.rst

```diff
@@ -1,59 +1,64 @@
-    +------------------+-------+------+-----+---+-----+---+---+
-    |OptionContainer   |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |PasswordInput     |no     |no    |yes  |no |yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Command           |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |SEPARATOR         |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Box               |yes    |yes   |no   |yes|yes  |yes|no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |SplitContainer    |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Table             |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |SPACER            |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |App               |yes    |yes   |yes  |yes|yes  |yes|no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |MainWindow        |no     |yes   |no   |yes|yes  |yes|no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Selection         |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |TIBERIUS_ICON     |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Icon              |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |TextInput         |no     |yes   |yes  |yes|yes  |yes|no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |ProgressBar       |no     |no    |no   |no |yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Font              |no     |no    |no   |yes|yes  |yes|no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |MultilineTextInput|no     |no    |yes  |no |yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Image             |no     |no    |no   |no |yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |Label             |no     |no    |yes  |yes|yes  |yes|no |
-    +------------------+-------+------+-----+---+-----+---+---+
-    |EXPANDING_SPACER  |no     |no    |no   |yes|yes  |no |no |
-    +------------------+-------+------+-----+---+-----+---+---+
+    +------------------+-----+-----+-----+-------+------+----+-----+
+    |    Component     | gtk |win32|cocoa|android|django|web | iOS |
+    +==================+=====+=====+=====+=======+======+====+=====+
+    |Icon              ||yes|||no| ||yes|||no|   ||no|  ||no|||no| |
```

Comment:
done in https://github.com/pybee/toga/pull/102/commits/b0a5fbf68e6b1d875c8767f325ffcd54e2e22688